flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/controller/RollbackTest.java

```diff
@@ -265,5 +265,28 @@ public void testRollback(
         assertTrue(reconciliationStatus.isLastReconciledSpecStable());
         assertEquals(ReconciliationState.DEPLOYED, reconciliationStatus.getState());
         assertNull(deployment.getStatus().getError());
+
+        if (deployment.getSpec().getJob() != null) {
```

Comment:
It seems that the tests do not cover the scenario which is suspending from `ROLLED_BACK` state.

Reply:
thanks, will add now

Reply:
fixed